src/Persistence/Migrations/20230606191818_AddBrokenBoolToItemAndRenamedUpgradeEndpointToRepair.cs

```diff
@@ -0,0 +1,125 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
You should probably update is_broken to true when rank < -1 in the migration

Reply:
done